Silicon/NXP/iMX6Pkg/Drivers/PlatformSmbiosDxe/PlatformSmbiosDxe.c

```diff
@@ -791,6 +902,9 @@ SysInfoUpdateSmbiosType1 (
   if (SystemUuid != NULL) {
     FreePool (SystemUuid);
   }
+  if (SkuNumber != NULL && SkuNumber != FixedPcdGetPtr (PcdSystemSkuNumber)) {

```

Comment:
nit - Add one more level of parentheses here
```suggestion
  if ((SkuNumber != NULL) && (SkuNumber != FixedPcdGetPtr (PcdSystemSkuNumber))) {
```